### ISSUE TITLE
Question 

### ISSUE THREAD

**Issue by Vlad0254** [ShikyoKira/Project-New-Reign---Nemesis-Main#559]
Hey ShikyoKira, great mod btw. Although I don't use animations like I once did (back when I ran many thousands of animations per load order). However my SSE Steam Load Order Guide does feature Nemesis as an option. 

I'm getting a little flack because some say you're no longer active but I see here that's not true. I also understand the Nemesis engine had a couple of updates in the last couple of days. So my question is where are those updates noted and is this a good link to post for users that have issues?

I don't use Github a ton other than downloading xEdit etc. so forgive me if the answer to these questions are obvious. I'm not even sure this is the right place to ask this question. 

thanks and keep up the great work.

Vlad 254
[https://steamcommunity.com/sharedfiles/filedetails/?id=1986067552](url)

**Comment by epinter** [comment]
@Vlad0254 
I didn't see any updates recently, where you found updates ? If you are talking about the "Update engine" feature, that process looks like to be completely local , based on mods and animations we have... I never see network access from Nemesis.

**Comment by Vlad0254** [comment]
@epinter
Yes, I misread the dates. I'm really just checking in while hoping to maybe hear from ShikyoKira. Either way Nemesis performs as advertised. I am watching the progress on Nemesis because as of right now it's part of my Skyrim SE load order guide which I am constantly updating so of course I try to keep up on the mods that are listed in the guide. I'll keep on keepin' on in the mean time. Appreciate the reply

**Comment by Vlad0254** [comment]
That said if anyone has any info regarding ShikyoKira and or plans for updates on Nemesis I'd appreciate it.

**Comment by epinter** [comment]
@Vlad0254 
I think you will find the plans and current development status on his patreon:
patreon.com/shikyokira

**Comment by Vlad0254** [comment]
thanks epinter

btw - I have a straw poll running right now NEMESIS and DAR are leading over FNIS as to which people one are using
https://strawpoll.com/okga39677
